src/web/sitemap.rs

```diff
@@ -54,26 +60,27 @@ pub fn sitemap_handler(req: &mut Request) -> IronResult<Response> {
     let query = conn
         .query(
             "SELECT crates.name,
+                    releases.target_name,
                     MAX(releases.release_time) as release_time
              FROM crates
              INNER JOIN releases ON releases.crate_id = crates.id
              WHERE 
                 rustdoc_status = true AND 
                 crates.name ILIKE $1 
-             GROUP BY crates.name
+             GROUP BY crates.name, releases.target_name
              ",
             &[&format!("{}%", letter)],
         )
         .unwrap();
 
     let releases = query
         .into_iter()
-        .map(|row| {
-            let time = row.get::<_, DateTime<Utc>>(1).format("%+").to_string();
-
-            (row.get(0), time)
+        .map(|row| SitemapRow {
+            crate_name: row.get(0),
+            target_name: row.get(1),
+            last_modified: row.get::<_, DateTime<Utc>>(2).format("%+").to_string(),
```

Comment:
using names instead of numbers helps avoid bugs - I know this is existing, but do you mind changing it since this code is changing anyway?
```suggestion
            crate_name: row.get("name"),
            target_name: row.get("target_name"),
            last_modified: row.get::<_, DateTime<Utc>>("release_time").format("%+").to_string(),
```